deploy_eks_cluster/README.md

```diff
@@ -0,0 +1,77 @@
+# EKS Cluster Deployment
+
+Deploy an EKS Cluster for Wazuh.
+
+## YAML files
+
+* `cluster.yml` should be used if the creation of the VPC and subnets is required.
+* `cluster_existing_vpc.yml` should be used if the VPC and subnets already exist.
+
+
+## AWS Resources
+
+The goal of this section is to simplify the creation of an EKS cluster.
+The execution of this guide will create 2 Cloudformation stacks with the following resources:
```

Comment:
Why are we using two cloudformation stacks, all the resources are duplicated on each stack. We need to explain the motivation behind this setup.

Reply:
Also, we need to describe what instances we are using and why.